desc/geometry/surface.py

```diff
@@ -758,17 +758,19 @@ def fun_jax(zeta_hat, theta, zeta):
             return offset_surface
 
 
-class ZernikeRZToroidalSection(Surface):
-    """A toroidal cross section represented by a Zernike polynomial in R,Z.
+class PoincareRZLSection(Surface):
```

Comment:
This deviates from our existing naming convention of `{Basis}{Parameterization}{Type}`. I think `ZernikeRZToroidalSection` was fine, I thought we were just going to add lambda to that?